### PR TITLE
DM-19890: Auxiliary Telescope in-dome software test

### PR DESCRIPTION
**Pull request by tribeiro**
This PR adds the notebooks developed during the visit to AT in 2019/05.

### CODE REVIEW

File: tribeiro/Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (couger01) [inline tribeiro/Untitled.ipynb]: Could you title this?

File: tribeiro/Untitled1.ipynb [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (couger01) [inline tribeiro/Untitled1.ipynb]: Could you title this?

File: tribeiro/Untitled2.ipynb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (couger01) [inline tribeiro/Untitled2.ipynb]: Could you title this?

**Review by couger01 — CHANGES_REQUESTED** [review]
Looks good to me, but I'd like the untitled notebooks to have some titles.

**Comment by tribeiro** [comment]
Thanks for spotting those. I removed then because they where pretty much useless and added a new one with measurement of the M1 pressure coefficients from the dataset.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**